src/managers/notification-manager.ts

```diff
@@ -69,17 +69,25 @@ class NotificationManager extends LitElement {
               ></mwc-button>
             `
           : ""}
+        <mwc-button
+          .label=${this.hass.localize("ui.notification_toast.dismiss")}
+          @click=${this.dismiss}
+        ></mwc-button>
```

Comment:
The dismissal action should be passed as an action from the caller. We should not just add a button on every toast.

Reply:
Ok, if I understand you, I added a default action to only dismiss toast, and now we will always have only one button. Take a look, please.